examples/basic/README.md

```diff
@@ -2,6 +2,8 @@
 
 A simple example that shows how to create a standard and a root key in an existing IBM Key Management Service (KMS) instance.
 
+The example is dependent on existing KMS instance guid, so to use this example you must have KMS Instance created before if you do not have it.
+
 The following resources are provisioned by this example:
  - Create a root key in the existing KMS instance.
  - Create a standard key in the existing KMS instance.
```

Comment:
```suggestion
The example provisions the following resources:
 - A root key in the existing KMS instance.
 - A standard key in the existing KMS instance.
```

Might as well fix up these list items. The lead-in already have the verb (provision), so we don't need it in the list items